apps/docs/src/app/platform/component-docs/platform-input/platform-input-header/platform-input-header.component.html

```diff
@@ -0,0 +1,9 @@
+<header>Input</header>
+<description>
+	The Input has the same features and functionality as the Input component in Fundamental NGX: Core. As such, 
```

Comment:
Let's a bit rework this description and let;s take it to our API review docs site